ci/Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM jenkinsci/blueocean
```

Comment:
Would you add some inline comments to explain what this docker container is used for for the community?